cmd/insert-user/main.go

```diff
@@ -53,21 +68,25 @@ func main() {
 
 	flag.Parse()
 
-	if len(os.Args) != 3 {
-		fmt.Fprintf(os.Stderr, "usage: %s <user-name> <@theirPublicKey.ed25519>\n", os.Args[0])
-		flag.Usage()
-		os.Exit(1)
-		return
+	// we require at least 5 arguments: <executable> + -name <val> + -key <val>
+	//                               1              2     3       4    5
```

Comment:
lol go fmt was not kind to this line :crying_cat_face:

Reply:
Sometimes It's more forgiving if you use `/*<newline> ... */`